Package.swift

```diff
@@ -1,11 +1,14 @@
-// swift-tools-version:5.5
+// swift-tools-version:5.6
```

Comment:
Dropped support for Swift 5.5, discussion here: https://github.com/airbnb/lottie-ios/pull/2152#discussion_r1303341991